lib/tilex/notifications/notifications.ex

```diff
@@ -73,4 +87,29 @@ defmodule Tilex.Notifications do
   def notifiers(notifiers_supervisor \\ NotifiersSupervisor) do
     notifiers_supervisor.children()
   end
+
+  defp schedule_report do
+    timezone =
+      case Timex.Timezone.get(@timezone) do
+        {:error, _} -> "America/Chicago"
```

Comment:
shouldn't we let it blow instead?

Reply:
I guess so? It just sucks that you won't know at compile time and only when it runs. That's why I put the fallback. But I agree the Elixir way is to probably just let it crash. The Genserver is supervised so it will just spin back up so no worries there. Does anyone else have any thoughts on letting it crash when it runs vs falling back to a default?

Reply:
I wish we could have a way to load that config values when the application boots, then we would leave for runtime the env valuations, fails fast but as soon as the app loads, so it's easier to catch wrong config

Reply:
Alright, still not done with everything, but I took care of this. Now if it can't parse the Timezone right at start (`Tilex.start/2`) it will blow up:

```terminal
** (Mix) Could not start application tilex: Tilex.start(:normal, []) returned an error: shutdown: failed to start child: Tilex.Notifications
    ** (EXIT) an exception was raised:
        ** (ArgumentError) argument error
            :erlang.send_after({:error, :invalid_date}, Tilex.Notifications, :generate_page_views_report)
            (tilex) lib/tilex/notifications/notifications.ex:104: Tilex.Notifications.schedule_report/0
            (tilex) lib/tilex/notifications/notifications.ex:37: Tilex.Notifications.init/1
```

Reply:
nice... thanks for putting the effort and sharing with us...

Reply:
I think that's the right approach as well. -chris